server/bleep/src/webserver/answer/prompts.rs

```diff
@@ -31,7 +31,7 @@ pub fn functions() -> serde_json::Value {
             },
             {
                 "name": "proc",
-                "description": "Read one or more files and extract the line ranges which are relevant to the search terms.",
+                "description": "Read one or more files and extract the line ranges which are relevant to the search terms. Do not proc more than 10 files at a time.",
```

Comment:
Why do we add this in the prompt rather than when consuming the results?

Reply:
The hypothesis is that it encourages the model to be selective about the files that it processes. Want to avoid situations where the model chooses to process every file in the path history. Have seen an example where it chose to process 54 files.